test/Microsoft.ServiceFabric.Services.Wcf.Tests/WcfRemotingListenerSettingsTest.cs

```diff
@@ -0,0 +1,98 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.ServiceFabric.Services.Remoting.V2.Runtime;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Wcf.Runtime;
+using Xunit;
+
+namespace Microsoft.ServiceFabric.Services.Wcf.Remoting.V2.Wcf.Runtime
+{
+    public abstract class WcfRemotingListenerSettingsTest
+    {
+        public WcfRemotingListenerSettingsTest()
```

Comment:
Empty public constructors shouldn't be needed, particularly in abstract classes.